git-helpers/tests/remote.rs

```diff
@@ -112,7 +112,10 @@ fn setup_entity(paths: &Paths, key: SecretKey) -> anyhow::Result<RadUrn> {
 fn setup_keystore(dir: &Path, key: SecretKey) -> anyhow::Result<()> {
     let mut keystore = keystore::FileStorage::<_, PublicKey, _, _>::new(
         &dir.join("librad.key"),
-        keystore::crypto::Pwhash::new(SecUtf8::from(PASSPHRASE)),
+        keystore::crypto::Pwhash::new(
+            SecUtf8::from(PASSPHRASE),
+            *keystore::crypto::KDF_PARAMS_PROD,
```

Comment:
As this is in the context of tests shouldn't it use test params?

Reply:
Yah, see my top-level comment. It should be TEST I think, but also shouldn't fail the test :sweat_smile: